### DISCUSSION TITLE
How to get CISCO WebEx Instructor Portal?

### DISCUSSION THREAD

**Discussion by carstenconrad1098** [community/community#138947] [GitHub Education] [answered]
**### I need to get CISCO WebEx Instructor portal. How to get it?**

**Comment by indrajithbandara** [comment, marked as answer]
After logging into your Netacad Instructor account, you will be able to see a **WebEx button** displayed in your profile section.

Once you submit a request through this, your _WebEx account will be created within a few minutes._

If the account is not created, you may contact the **_NetAcad support desk or open a ticket_**. They will ensure that your WebEx Instructor account is set up within three business days.

**Reply by carstenconrad1098** [reply]
Thank you.

**Reply by indrajithbandara** [reply]
Welcome

**Reply by carstenconrad1098** [reply]
> After logging into your Netacad Instructor account, you will be able to see a **WebEx button** displayed in your profile section.
> 
> Once you submit a request through this, your _WebEx account will be created within a few minutes._
> 
> If the account is not created, you may contact the **_NetAcad support desk or open a ticket_**. They will ensure that your WebEx Instructor account is set up within three business days.

Thank you very much your support

**Comment by indrajithbandara** [comment]
Are you CISCO Instructor?

**Reply by carstenconrad1098** [reply]
Yes.

**Comment by indrajithbandara** [comment]
Which Academy do you teach?

**Reply by carstenconrad1098** [reply]
German Supervisor NetAcad Academy
